Tests/FluentAssertions.Specs/Collections/GenericCollectionAssertionsSpecs.cs

```diff
@@ -1284,6 +1284,118 @@ public void When_asserting_the_items_in_ay_collection_are_not_ordered_using_an_i
 
         #endregion
 
+        #region Then be in order
+
+        [Fact]
+        public void When_asserting_subsequence_by_then_be_in_ascending_order_it_should_pass()
```

Comment:
🤔 Try to keep the names of test functional and never refer to the literal name of a method or API, e.g.

`When_the_collection_is_ordered_according_to_the_subsequent_ordering_assertion_it_should_succeed`